libcontainer/container_linux.go

```diff
@@ -1192,6 +1193,15 @@ func (c *linuxContainer) criuNotifications(resp *criurpc.CriuResp, process *Proc
 				logrus.Error(err)
 			}
 		}
+	case notify.GetScript() == "orphan-pts-master":
+		scm, err := syscall.ParseSocketControlMessage(oob)
+		if err != nil {
+			return err
+		}
+		fds, err := syscall.ParseUnixRights(&scm[0])
+
+		process.consoleChan = make(chan *os.File, 1)
+		process.consoleChan <- os.NewFile(uintptr(fds[0]), "console")
```

Comment:
I'm not sure if we want to land this before or after #1356 -- but we need to make sure this is handled properly given the changes in #1356.